### DISCUSSION TITLE
Vue Router terminal warning

### DISCUSSION THREAD

**Discussion by DamianGlowala** [nuxt/nuxt#17562] [Questions] [answered]
Happens on every page refresh. Shall it be reported in a different repo or it might be a Nuxt-related issue?

![image](https://user-images.githubusercontent.com/48835293/169554169-e119a89c-e8fb-455a-9628-2c1e6a5e61dc.png)

**Comment by pi0** [comment, marked as answer]
This should be mitigated once we enable serve-placeholder (https://github.com/unjs/nitro/issues/94). After middleware, static, and server routes and before hitting SSR (that vue-router is loaded), if the path contains a known extension such as `.js` and is 404, we serve a placeholder.

I think you can fix it by removing service worker manually from dev tools.

**Reply by DamianGlowala** [reply]
Unregistering service worker in the dev tools worked. Thanks! :)
